gapir/cc/archive_replay_service.cpp

```diff
@@ -25,8 +25,15 @@ namespace gapir {
 std::unique_ptr<ReplayService::Payload> ArchiveReplayService::getPayload(
     const std::string&) {
   std::fstream input(mFilePrefix, std::ios::in | std::ios::binary);
+  if (!input) {
```

Comment:
Why are you using NULL instead of nullptr? And in the "if" statement, why are you using "!" instead of "nullptr == "?

Reply:
"input" is an fstream, not a pointer. The ! operator is defined for std::fstreams to return if the stream is good or not,

Reply:
http://www.cplusplus.com/reference/ios/ios/operator_not/